go.mod

```diff
@@ -5,8 +5,8 @@ go 1.17
 require (
 	github.com/Azure/azure-sdk-for-go v65.0.0+incompatible
 	github.com/Azure/go-autorest/autorest v0.11.24
-	github.com/notaryproject/notation-core-go v0.1.0-alpha.3
-	github.com/notaryproject/notation-go v0.10.0-alpha.3.0.20220819145229-a9fb055b8d92
+	github.com/notaryproject/notation-core-go v0.1.0-alpha.3.0.20220929020818-b5df54c7465b
```

Comment:
`notation-core-go v0.1.0-alpha.4` has been released. Let's update to that.

Reply:
Maybe we should wait for notation-go alpha.4 release and update the dependency accordingly.

Reply:
We can bump it in later PRs.